salt/metalk8s/orchestrate/bootstrap_step2.sls

```diff
@@ -0,0 +1,37 @@
+Bootstrap CA:
+  salt.state:
+    - tgt: bootstrap
+    - saltenv: metalk8s-2.0
```

Comment:
Are we sure about hardcoding the `2.0` version?

Reply:
Done, using `{{ saltenv }}` as suggested by @TeddyAndrieux